templates/docs/examples/patterns/navigation/_script.js

```diff
@@ -1,81 +1,61 @@
-/**
-  Toggles visibility of given subnav by toggling is-active class to it
-  and setting aria-hidden attribute on dropdown contents.
-  @param {HTMLElement} subnav Root element of subnavigation to open.
-  @param {Boolean} open indicate whether we want to open or close the subnav.
-*/
-function toggleSubnav(subnav, open) {
-  if (open) {
-    subnav.classList.add('is-active');
-  } else {
-    subnav.classList.remove('is-active');
-  }
-
-  var toggle = subnav.querySelector('.p-subnav__toggle');
-
-  if (toggle) {
+(function () {
```

Comment:
Do we need a full second script? Functionality should be the same, just the class names were changed. Would it be possible to extract class names to some params and have one script serve both examples?

Reply:
Good point - initially just deprecated it so as not to mess with the deprecated example at all, and then created a refactored version, but yeah, it'll totally work for both.